### ISSUE TITLE
Travis build breaking (new cause)

### ISSUE THREAD

**Issue by ptaoussanis** [http-kit/http-kit#262]
https://travis-ci.org/http-kit/http-kit/builds/112409573

```
lein test :only org.httpkit.ws-test/test-message-executed-in-order
FAIL in (test-message-executed-in-order) (ws_test.clj:167)
expected: (= "true" (.getMessage client))
  actual: (not (= "true" nil))

<...> ; 8 more like this

Ran 65 tests containing 6834 assertions.
9 failures, 0 errors.
Tests failed.
```

The test itself:
```clojure
;;; make sure Message ordering is guaranteed
(deftest test-message-executed-in-order
  (doseq [_ (range 1 5)]
    (let [client (WebSocketClient. "ws://localhost:4348/order")]
      (doseq [id (range 1 10)]
        (.sendMessage client (pr-str {:id id}))
        (is (= "true" (.getMessage client))))
      (.close client)))
  (doseq [_ (range 1 5)]
    (let [client (WebSocketClient. "ws://localhost:4348/order")]
      ;; 10 concurrent message
      (doseq [id (range 1 10)]
        (.sendMessage client (pr-str {:id id})))
      (doseq [_ (range 1 10)]
        (is (= "true" (.getMessage client))))
      (.close client))))
```

Haven't been able to reproduce a test failure locally, only on Travis - so guessing it may be another environmental thing. Will need to dig.

**Comment by mhjort** [comment]
I think this PR #265 solves the problem

**Comment by ptaoussanis** [comment]
Closing, confirmed that this is resolved by #265 - thanks!
